doc/howto/using_the_api.rst

```diff
@@ -41,6 +41,21 @@ on NAV installations that do not have SSL enabled, you are potentially giving
 everyone access to the data.
 
 
+JSON Web Tokens
+------------------
+JSON Web Tokens (JWTs) must be :doc:`configured <../reference/jwt>` before they can be used.
+
+Once configured, you can use tokens issued by your configured issuers in almost the same way
+as "normal" tokens::
```

Comment:
Nitpick: "normal" is not a good name. Rename to `classic` without any scare-quotes and maybe add a link to docs about the classic kind.

Reply:
did done this